src/generators/genlua.ml

```diff
@@ -278,7 +277,56 @@ let rec is_string_type t =
 	| _ -> false
 
 let is_string_expr e = is_string_type e.etype
-(* /from genphp *)
+
```

Comment:
This block is all the helper functions that deal with extracting values out of the v_meta fields for a local variable, or building or finding an index.